aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -1567,5 +1571,42 @@ bool use_fast_accum) {
 
 }
 
+Tensor _grouped_mm_cuda(const Tensor& mat_a, const Tensor& mat_b,
+const std::optional<at::Tensor>& offs,
+const std::optional<at::Tensor>& bias,
+std::optional<c10::ScalarType> out_dtype) {
+#ifndef USE_ROCM
+  bool allowed_device = _scaled_mm_allowed_device();
+  TORCH_CHECK(allowed_device, "torch._scaled_mm is only supported on CUDA devices with compute capability >= 9.0 or 8.9, or ROCm MI300+");
```

Comment:
nit: this will never be printed on ROCm?
Does >= 8.9 or 9.0 mean >= 8.9?
Would this also be expected to work on Geforce Blackwell, SM 12.0?

Reply:
ROCM prints an error in the #else block